jupyter_server/extension/application.py

```diff
@@ -181,8 +181,8 @@ def _validate_extension_name(self, proposal):
         ServerApp,
     ]
 
-    aliases = aliases
-    flags = flags
+    # aliases = aliases
+    # flags = flags
```

Comment:
Whenever I see commented out code like this, I assume this was some temporary measure? Could you either add a comment explaining why this code is still here, or remove it?

Reply:
Aliases and flags are now empty by default in ExtensionApps